escoba/src/views/Board.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="board">
+    <div v-if="getGameDataLoaded">
```

Comment:
seems like this is the only level this v-if should be needed no?

Reply:
probably really only needed in app, or yeah here.  started adding them at work for redundancy and because it was taking me a while to figure out exactly where needed.